frontend/src/App.tsx

```diff
@@ -65,6 +65,13 @@ export default function App() {
             .then(closeModalUpdate)
     }
 
+    function handleDelete(event: React.MouseEvent<HTMLButtonElement>) {
+        event.preventDefault()
+        void axios.delete("/api/character_cards/" + id)
```

Comment:
if this void doesn't have a specific reason to be, remove it for sonar

Reply:
Thank you for your suggestion, Enrico. To be honest, I don't really understand [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) yet. So, I'm learning now how to use that 🤓 I will make improvements to the cods on my next PR 🚀